llms/mlx_lm/lora.py

```diff
@@ -177,15 +177,15 @@ def load_dataset(args):
 
     # Freeze all layers other than LORA linears
     model.freeze()
-    for l in model.model.layers[len(model.model.layers) - args.lora_layers :]:
+    for l in model.model.layers[len(model.model.layers) - args.lora_layers:]:
         l.self_attn.q_proj = LoRALinear.from_linear(l.self_attn.q_proj)
         l.self_attn.v_proj = LoRALinear.from_linear(l.self_attn.v_proj)
         if hasattr(l, "block_sparse_moe"):
             l.block_sparse_moe.gate = LoRALinear.from_linear(l.block_sparse_moe.gate)
 
-    p = sum(v.size for _, v in tree_flatten(model.parameters())) / 10**6
+    p = sum(v.size for _, v in tree_flatten(model.parameters())) / 10 ** 6
     print(f"Total parameters {p:.3f}M")
-    p = sum(v.size for _, v in tree_flatten(model.trainable_parameters())) / 10**6
+    p = sum(v.size for _, v in tree_flatten(model.trainable_parameters())) / 10 ** 6
```

Comment:
Could you run the pre-commit hooks? I think the formatter liked it more the way it was.